src/components/Timeline/TimelineNote.tsx

```diff
@@ -97,7 +97,7 @@ export const TimelineNote: React.FC<TimelineNoteProps> = props => {
         <CardContent className={classes.cardContent}>
           <Typography
             dangerouslySetInnerHTML={{
-              __html: message.replace("\n", "<br />")
+              __html: message.replace(/\n/, "<br />")
```

Comment:
Won't this replace only the first occurrence? 

``` 
"foobar \n pops \n".replace(/\n/, "<br />")
```
will do:
```
"foobar <br /> pops 
"
```

`replaceAll` ?